bpdv3/Assets/Scripts/Floor/TileController.cs

```diff
@@ -43,6 +45,7 @@ void Update()
             }
         }
 
+        // ** The 12th beat.
         if (SceneController.instance.scrollBeatCount == 12)
```

Comment:
I see the comment but still a magic number, if you define it as constant you wont need comment. Use comments as little as possible